gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace GmodNET
+{
+    /// <summary>
+    /// Redirects Console's output to game's console
+    /// </summary>
+    public class GameConsoleWriter : TextWriter
+    {
+        public override string NewLine => "\n";
+        public override Encoding Encoding => Encoding.UTF8;
+
+#pragma warning disable CA2101
+        [DllImport("tier0", CallingConvention = CallingConvention.Cdecl)]
+        private static extern void Msg([MarshalAs(UnmanagedType.LPUTF8Str)] string msg);
+
+        public override void Write(string value)
+        {
+            Msg(new string(value));
```

Comment:
Are we crating string from string? What for?

Reply:
much more easier check for `null`, `Msg(null)` crashes

Reply:
I should replace it with
```cs
if(value!=null){
    Msg(value);
}
```
Right?

Reply:
Right

Reply:
overwrited in https://github.com/GmodNET/GmodDotNet/pull/70#discussion_r532084708 resolved in https://github.com/GmodNET/GmodDotNet/pull/70/commits/415c5f62feca5cd0d2a75590c56c2174f8fe0edd